server/controllers/admin/donors/getDonors.ts

```diff
@@ -0,0 +1,43 @@
+import { NextFunction, Request, Response } from 'express';
+import {
+  Donation, Donor, sequelize,
+} from '../../../database/models';
+import { CustomError } from '../../../utils';
+import { donorsQuerySchema } from '../../../utils/validation';
+
+const getDonors = async (req:Request, res:Response, next:NextFunction) => {
+  try {
+    const { page = 1, limit = 6 } = req.query;
+    await donorsQuerySchema.validateAsync(req.query);
```

Comment:
short the two lines for 
const { page = 1, limit = 6 } = donorsQuerySchema.validateAsync(req.query);